packages-common/pluggable-widgets-generator/generators/app/templates/pluggable/web/emptyTemplateTs/src/WidgetName.tsx.ejs

```diff
@@ -0,0 +1,14 @@
+import { Component, ReactNode, createElement } from "react";
+import { HelloWorldSample } from "./components/HelloWorldSample";
+import { hot } from "react-hot-loader/root";
+import { <%= widgetName %>ContainerProps } from "../typings/<%= widgetName %>Props";
+
+import "./ui/<%= widgetName %>.css";
+
+class <%= widgetName %> extends Component<<%= widgetName %>ContainerProps> {
+    render(): ReactNode {
+        return <HelloWorldSample sampleText={this.props.sampleText ? this.props.sampleText : "World"} />;
```

Comment:
Coalesce operator..

Reply:
This operator will only work with null values

Reply:
Feedback reference included in a new story, so will be picked up in a follow up PR.